frontend/src/my-knolist/Sidebar.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { Sidenav, Icon, Nav } from "rsuite";
+import { Link, useLocation } from "react-router-dom";
+
+function Sidebar() {
+  let first = false;
+  let second = false;
+  let third = false;
+  const location = useLocation()["pathname"];
+
+  if (location === "/my-projects") first = true;
+  else if (location === "/shared") second = true;
+  else if (location === "./archived") third = true;
```

Comment:
I think archived is not displaying as active because here you have `location === "./archived", you probably just need to get rid of that first dot.
```suggestion
  else if (location === "/archived") third = true;
```